pkgs/applications/editors/vim/plugins/non-generated/ssr/default.nix

```diff
@@ -0,0 +1,6 @@
+{
+  vimPlugins,
+}:
+vimPlugins.ssr-nvim.overrideAttrs {
```

Comment:
Q: is it intended that `vimPlugins.ssr` overrides `vimPlugins.ssr-nvim`?

At first glance, it looks like the override may have been missed when renaming the plugin.

Should this PR "fix" that by renaming `ssr` and adding an alias?

Reply:
When I saw the diff earlier, this is what popped up as weird to me. I think we just need to fix the override.